.circleci/config.yml

```diff
@@ -133,7 +133,7 @@ jobs:
           command: npx athloi version ${CIRCLE_TAG}
       - run:
           name: NPM publish
-          command: npx athloi publish -- --access=public
+          command: npm publish --workspaces
```

Comment:
**question**: will this have the same problem re: `file:` dependencies that Page Kit had?